vspace/src/main/java/edu/asu/diging/vspace/core/model/impl/Exhibition.java

```diff
@@ -29,6 +30,10 @@
     private ISpace startSpace;
 
     private String title;
+    
+    private ExhibitionModes mode;
```

Comment:
you want to make sure the enum is mapped as a string, so we don't have to worry about changing the order of the values.
`@Enumerated(EnumType.STRING)`